state-chain/custom-rpc/src/lib.rs

```diff
@@ -791,29 +814,50 @@ where
 
 	fn cf_pool_price(
 		&self,
-		from_asset: RpcAsset,
-		to_asset: RpcAsset,
+		from: RpcAsset,
+		to: RpcAsset,
```

Comment:
Also can you revert the name changes of this function, for consistently with the trait definition.